osclib/stagingapi.py

```diff
@@ -760,15 +764,27 @@ def rm_from_prj(self, project, package=None, request_id=None,
 
         self._remove_package_from_prj_pseudometa(project, package)
         subprj = self.map_ring_package_to_subject(project, package)
+        if self._supersede:
+            self.is_package_disabled(subprj, package, store=True)
         delete_package(self.apiurl, subprj, package, force=True, msg=msg)
 
         for sub_prj, sub_pkg in self.get_sub_packages(package):
             sub_prj = self.map_ring_package_to_subject(project, sub_pkg)
+            if self._supersede:
```

Comment:
My thought at this point is two-fold:

1. Lets prioritize #1002 (real OBS to test against) so we have some way to walk through all the possible workflows and verify since this is some fairly scary code.
2. It may make sense for me to revisit my prototype `ring` command (#848) and finish it as that would provide supplemental information that would make it crystal clear which packages should be enabled rather than the somewhat wishy-washy alternatives (looking at previous build settings).